### ISSUE TITLE
[TSDoc> Playground]: Expand/Collapsed code button is not accessible through the keyboard.

### ISSUE THREAD

**Issue by vagpt** [microsoft/rushstack-websites#251]
**Test Environment:**
Version: Version 24H2 (OS Build 26231.5000)
Browser Version: 127.0.2638.2 (Official build) dev (64-bit)
URL: https://tsdoc.org/play/

**Pre-Requisite:**
Navigate through keyboard only.

**Repro Steps:**
1. Open the above URL on the EDGE Dev browser.
2. Press the tab key and navigate to the 'PlayGround' option and press enter key to activate it.
3. Press the tab key and navigate to code editor section.
4. Now try to access the expand or collapse code button and observe the issue.

**Actual Result:**
While navigating the code editor section, Expand or collapse code button is not accessible through the keyboard. As on pressing tab, shift+tab key or any other short keys it is not accessible.

**Observation:**
Using mouse click button is accessible.

**Expected Result:**
Expand/Collapsed code button under code editor section should be accessible through the keyboard.

**User Impact:**
It will impact the keyboard user as they will not be able to access the button due to this user will not able to expand or collapsed the code section.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/keyboard

Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment.

https://github.com/microsoft/rushstack-websites/assets/95913748/fbb1263f-e5e6-4ac5-b8ed-eb43654e5505

![Expand Collapsed code button is not accessible through the keyboard](https://github.com/microsoft/rushstack-websites/assets/95913748/ad4b27a8-71c0-48f3-b27f-14a70a699d51)
